src/Boot/BootstrapManager.php

```diff
@@ -385,26 +386,22 @@ public function bootstrapValidate($phase)
      * @param \Consolidation\AnnotatedCommand\AnnotationData $annotationData
      *   Optional annotation data from the command.
      *
-     * @return bool
```

Comment:
we should keep returning TRUE upon success, for backward compat. can be marked as deprecated.